knowledge-base/window-does-not-display-child-items-data.md

```diff
@@ -0,0 +1,87 @@
+---
+title: The Window does not display its child items data
+description: How to ensure displaying the child items data within a Window.
+type: troubleshooting
+page_title: The Window does not display its child items data
+slug: window-kb-does-not-display-child-items-data
+position: 
+tags: window, display, child, items, empty, data
+ticketid: 1602103
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Window for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+If you have a component, such as DropDownList, nested within a Telerik Window, you may see a situation where the DropDownList items appear blanked. In scenarios like this, the Window does not display its child component data.
+
+## Cause\Possible Cause(s)
+The reason for the issue is that the child component receives its data after the Window becomes visible.
+
+## Solution
+You need to refresh the Window through its `Refresh()` method to have the UI update and the data displayed. 
+
+### Example
```

Comment:
Introducing the example as a part of the solution is enough, you don't need to separate it from the solution through another heading.
```suggestion
```